packages/next-auth/src/providers/discord.ts

```diff
@@ -31,15 +32,18 @@ export default function Discord<P extends DiscordProfile>(
     userinfo: "https://discord.com/api/users/@me",
     profile(profile) {
       if (profile.avatar === null) {
-        const defaultAvatarNumber = parseInt(profile.discriminator) % 5
+        const defaultAvatarNumber =
```

Comment:
Please revert changes in this file. Changes under next-auth are not going to be merged because we have v4 branch for that ☝️

Reply:
@Syjalo are you still interested in getting this merged? :blush:

Reply:
Sorry, I missed the review